api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -316,6 +316,26 @@ message Pass {
   uint64 uplink_center_frequency_hz = 10;
 }
 
+// Data uploaded from the ground station during a pass.
+message TelemetryData {
+  // URL of the data. This URL is valid for one hour after being returned.
+  string url = 1;
+
+  // The type of data available at the above URL.
+  enum DataType {
+    // Raw IQ data.
+    RAW = 0;
+
+    // Demodulated IQ data.
+    DEMODULATED = 1;
+
+    // Decoded data.
+    DECODED = 2;
+  }
+  // The status of this plan.
+  DataType data_type = 2;
+}
```

Comment:
I think we'll want to return zip archives of many decoded packets in some cases - do you think we'll later add an enum of archivetype?

Reply:
I'm hoping that the storage URL we return can return a content-type header when it's fetched so we don't need to include specific data types here.

Reply:
Having to parse a content-type will be much harder to use I think. But either way, it should be simple enough to add later.